services-matching/src/main/java/com/mapbox/api/matching/v5/MapboxMapMatching.java

```diff
@@ -338,6 +364,28 @@ public static Builder builder() {
     private String[] waypointNames;
     private String[] approaches;
 
+    /**
+     * Use GET method to request data (default).
```

Comment:
Minor comment - Shouldn't this be `POST` instead of `GET`? It seems it got reversed with the `get` method below 👇